Tests/XcodeprojTests/FunctionalTests.swift

```diff
@@ -157,6 +157,7 @@ func XCTAssertXcodeBuild(project: AbsolutePath, file: StaticString = #file, line
         let swiftLibraryPath = resolveSymlinks(swiftCompilerPath).appending(components: "..", "..", "lib", "swift", "macosx")
         if localFileSystem.exists(swiftCompilerPath) {
             stream <<< "SWIFT_LIBRARY_PATH = " <<< swiftLibraryPath.asString <<< "\n"
+            stream <<< "TOOLCHAIN_DIR = " <<< resolveSymlinks(swiftCompilerPath).appending(components: "..", "..", "lib", "swift", "pm").asString <<< "\n"
```

Comment:
`resolveSymlinks(swiftCompilerPath).appending(components: "..", "..", "..").asString` would fix the build error. However, if we do not use the `PackageDescription` built during the test, we will be in trouble when the specification of `PackageDescription` changes in the future.

Reply:
You're right, my bad.